flocker/node/agents/testtools.py

```diff
@@ -82,73 +36,24 @@ def setUp(self):
     return Tests
 
 
-def get_client(provider):
+class INovaVolumeManagerTestsMixin(object):
```

Comment:
Oops, yes.  I don't want to screw around with git moving hunks around when they're all going to land in master today anyway, though.